quiz.js

```diff
@@ -0,0 +1,36 @@
+var button = document.getElementById("grow");
+var inputHeight = document.getElementById("height");
+var inputChar = document.getElementById("char");
+
+button.addEventListener("click", checkMe);
+inputChar.addEventListener("keyup", enterKey);
+inputHeight.addEventListener("keyup", enterKey);
+function enterKey(event){
+    if(event.keyCode === 13){
+      checkMe();
+    }
+};
+
+function checkMe () {
+	if (inputHeight.value === "" || inputChar.value === "") {
+    	alert("Both Fields Required!");
+	} else {
+      	var tree = {
+        	height: inputHeight.value,
+        	char: inputChar.value,
+        };
+     	buildTree(tree);
+    }
+};
+
+function buildTree(treeMe) {
+	var char = treeMe.char;
+	var space = treeMe.height - 1;
+  var charString = "";
```

Comment:
Make sure your tabbing is correct and uniform. Some places are two, and other are four.